samples/Thinktecture.Blazor.Sample/Pages/ScreenWakeLock.razor.cs

```diff
@@ -0,0 +1,43 @@
+using Microsoft.AspNetCore.Components;
+using Thinktecture.Blazor.ScreenWakeLock;
+
+namespace Thinktecture.Blazor.Sample.Pages;
+
+public partial class ScreenWakeLock
+{
+    [Inject] private IScreenWakeLockService _screenWakeLockService { get; set; } = default!;
+
+    private bool _wakeLockRequested;
+    private bool _isSupported;
+
+    protected override async Task OnInitializedAsync()
+    {
+        _isSupported = await _screenWakeLockService.IsSupportedAsync();
+        _screenWakeLockService.WakeLockReleased = () =>
+        {
+            _wakeLockRequested = false;
```

Comment:
It looks like this doesn't rerender the page. When the lock is released (e.g., due to a tab change), the checkbox remains active.